.github/workflows/release.yml

```
@@ -0,0 +1,36 @@
+---
+name: release-helm-chart
+on:
+  push:
+    tags:
+    - '*'
+jobs:
+  release-helm-chart:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pull the master branch
+        uses: actions/checkout@v2
+        with:
+          path: chart/
+      - name: Pull the gh-pages branch
+        uses: actions/checkout@v2
+        with:
+          path: repository/
+          ref: gh-pages
+      - uses: azure/setup-helm@v1
+      - name: Pack the current chart version
+        run:  helm package -d repository/ chart/anycable-go
+      - name: Reindex Helm repository
+        run: >-
+          helm repo index
+          --url https://helm.anycable.io/
+          --merge repository/index.yaml
+          repository/
+      - name: Commit & Push gh-pages with a new chart
+        run: |
+          cd repository/
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add --all
+          git commit -m "Release ${GITHUB_REF##*/} to gh-pages"
+          git push origin gh-pages
```

I believe that we can use one of existing actions for that (there are plenty of them: [1](https://github.com/marketplace/actions/github-pages), [2](https://github.com/marketplace/actions/deploy-to-github-pages), [3](https://github.com/marketplace/actions/github-pages-action)). For example:
```suggestion
      - name: Deploy pages
        uses: crazy-max/ghaction-github-pages@v2
        with:
          build_dir: repository
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```